app/src/main/java/com/nervousfish/nervousfish/modules/constants/Constants.java

```diff
@@ -37,6 +39,10 @@
      */
     private Constants(final IServiceLocator serviceLocator) {
         this.androidFilesDir = serviceLocator.getAndroidFilesDir();
+        //noinspection StringConcatenationMissingWhitespace because this is a file path
+        this.databaseContactsPath = this.androidFilesDir + Constants.DB_CONTACTS_PATH;
+        //noinspection StringConcatenationMissingWhitespace because this is a file path
```

Comment:
To suppress Intellij warnings, I'm thinking about removing them, they're not very aesthetic